docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+services:
+  webpack:
+    container_name: dim-webpack
+    build:
+      context: ./docker-compose/webpack
```

Comment:
I suggest pairing an `image:` directive here underneath `build`. When done in this way, it will use the value to name and tag the image the container is started from, and helps with any cleanup automation we may add later.